### ISSUE TITLE
Conflict in complete new laravelproject

### ISSUE THREAD

**Issue by svenssontobias** [AidynMakhataev/laravel-surveyjs#43]
In a complete new Laravel project (created and fully updated 7feb 2021)
After running
composer require aidynmakhataev/laravelsurveyjs

 Problem 1
    - Root composer.json requires aidynmakhataev/laravelsurveyjs ^0.3.1 -> satisfiable by aidynmakhataev/laravelsurveyjs[0.3.1].
    - aidynmakhataev/laravelsurveyjs 0.3.1 requires illuminate/support ~5 -> found illuminate/support[v5.0.0, ..., 5.8.x-dev] but it conflicts with another require.

Any advice on how to solve this?

**Comment by marcosrocha85** [comment]
+1

**Comment by marcosrocha85** [comment]
@svenssontobias Here's what I did.

1. Clone this repository to your hard drive.
2. Add laravel/helpers (`composer require laravel/helpers`) to aidynmakhataev/laravelsurveyjs.
3. Update laravelsurveyjs composer.json from:
```
"require": {
    "illuminate/support": "~5",
    "cviebrock/eloquent-sluggable": "^4.0"
},
"require-dev": {
    "phpunit/phpunit": "~6.0",
    "orchestra/testbench": "~3.0"
},
```
to:
```
"require": {
    "illuminate/support": "^8",
    "cviebrock/eloquent-sluggable": "^8.0"
},
"require-dev": {
    "phpunit/phpunit": "^9.3.3",
    "orchestra/testbench": "^3.0"
},
```
4. In your project, add the key to composer.json.
```
"repositories": [
    {
      "type": "path",
      "url": "../path-you-cloned-this-project"
    }
],
```
5. Add the package manually to your project's composer.json require:
`"aidynmakhataev/laravelsurveyjs": "@dev",`
6. Do a `composer update`

**Comment by marcosrocha85** [comment]
It's good to mention that this is a workaround. The good practice is to create a merge request updating the project and creating a new version compatible to Laravel 8 (and minor versions as well).

**Comment by AidynMakhataev** [comment]
https://github.com/AidynMakhataev/laravel-surveyjs/pull/48 pull request was merged, you can now upgrade to 1.0.0 version with support Laravel 8
